tests/integration/admin_test.py

```diff
@@ -80,11 +84,161 @@ def test_update_groups(client: AtlanClient, group1: CreateGroupResponse):
     client.update_group(group)
 
 
-# TODO: users and then group membership
+def _get_user1_from_list(users: list[AtlanUser]) -> Optional[AtlanUser]:
+    for user in users:
+        if user.email == USER_EMAIL1.lower():
+            return user
+    return None
+
+
+def _get_user2_from_list(users: list[AtlanUser]) -> Optional[AtlanUser]:
+    for user in users:
```

Comment:
Perhaps change to:
`return next(
        (user for user in users if user.email == USER_EMAIL2.lower()), None
    )`